src/app/mock-tasks.ts

```diff
@@ -0,0 +1,13 @@
+import { Tasks } from "./tasks";
+export const tasks: Tasks[] = [
+    {id: 1, title: "Task 1", description: "Description for Task 1"},
+    {id: 2, title: "Task 2", description: "Description for Task 2"},
+    {id: 3, title: "Task 3", description: "Description for Task 3"},
+    {id: 4, title: "Task 4", description: "Description for Task 4"},
+    {id: 5, title: "Task 5", description: "Description for Task 5"},
+    {id: 6, title: "Task 6", description: "Description for Task 6"},
+    {id: 7, title: "Task 7", description: "Description for Task 7"},
+    {id: 8, title: "Task 8", description: "Description for Task 8"},
+    {id: 9, title: "Task 9", description: "Description for Task 9"},
+    {id: 10, title: "Task 10", description: "Description for Task 10"},
+]
```

Comment:
instead of hard-coded list we can use a mock server and we can set this data there and then get this data through an API call